robocop/config.py

```diff
@@ -82,6 +82,90 @@ def validate_regex(pattern: str) -> Pattern:
         raise ConfigGeneralError(f"Provided regex pattern {pattern} failed to be compiled")
 
 
+def resolve_relative_path(orig_path, config_dir: Path, ensure_exists: bool):
+    path = Path(orig_path)
+    if path.is_absolute():
+        return orig_path
+    resolved_path = config_dir / path
+    if not ensure_exists or resolved_path.exists():
+        return str(resolved_path)
+    return orig_path
+
+
+class ArgumentFileParser:
```

Comment:
These changes are in the scope of another PR #774 and should be removed from here.

Reply:
Rebased after merging #774